inc/blocks/rest_api/namespace.php

```diff
@@ -46,13 +46,25 @@ function init() : void {
 				],
 				'days' => [
 					'description' => __( 'Number of days worth of data to get.', 'altis-analytics' ),
-					'type' => 'number',
-					'default' => 7,
+					'type' => 'string',
+					'default' => 'P7D',
 				],
 				'offset' => [
 					'description' => __( 'Number of days to offset data by. For example 3 means get data from before 3 days ago.', 'altis-analytics' ),
-					'type' => 'number',
-					'default' => 0,
+					'type' => 'string',
+					'default' => 'P0D',
```

Comment:
A duration of 0 days doesn't seem valid here. Offset here was used as a number of days previous to get data for but if we're using `start` and `end` instead it becomes redundant. We want to remove the use of durations on the backend and just work with start and end dates instead.

Reply:
Would we not be doing away with the `offset` and `days`?

Reply:
We would be yes - but specifically on the backend, in the JS we want to standardise on using the period / duration notation.

I need to more fully review where things are at in terms of the API endpoints and how they're used - it's more involved than I first thought when I asked you to work on it.